### PR TITLE
Always show ex logo at sponsored gyms

### PR DESCRIPTION
**Pull request by hammydown4325**
Made it so sponsored gyms are always shown as EX Raid Compatible. Some are not showing up for some people.

### CODE REVIEW

**Review by CalamityJames — APPROVED** [review]
The code seems fine, however I cannot test this in my real world environment as we do not have sponsored gyms here.

File: static/js/map.js [hunk 1]
```diff
@@ -904,7 +904,7 @@ function getGymMarkerIcon(item) {
         teamStr = gymTypes[item['team_id']] + '_' + level
     }
     var exIcon = ''
-    if ((((park !== 'None' && park !== undefined && onlyTriggerGyms === false && park) || triggerGyms.includes(item['gym_id'])) && (noExGyms === false))) {
+    if ((((park !== 'None' && park !== undefined && onlyTriggerGyms === false && park) || item['sponsor'] > 0 || triggerGyms.includes(item['gym_id'])) && (noExGyms === false))) {
```
Comment (Glennmen) [inline static/js/map.js]: How does it handle if `item['sponsor']` is not set/undefined?

**Comment by hammydown4325** [comment]
Good thinking. We could use more of that type of logic throughout the map.js file anyway let me add  an !== Undefined real quick

**Comment by Glennmen** [comment]
@iroken222 Can you fix the merge conflict, thanks.

**Comment by Glennmen** [comment]
This seems to have already been included in PR #54 so this PR can be closed.

**Pull request closed without merging**